conjure-undertow-lib/src/main/java/com/palantir/conjure/java/undertow/lib/ServiceContext.java

```diff
@@ -38,13 +41,26 @@ public SerializerRegistry serializerRegistry() {
         return serializerRegistry;
     }
 
+    /**
+     * {@link ServiceInstrumenter} to apply metric instrumentation to exposed services.
+     */
+    public ServiceInstrumenter serviceInstrumenter() {
```

Comment:
Can we make this optional such that not instrumenting is the empty case.

Reply:
It's already a no-op by default?

Reply:
Yes, but my point was that the Optional pattern is imho more readable than the NO_OP one. But to be fair, that's just a matter of preference.